docker-for-windows/wsl-tech-preview.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Docker Desktop WSL 2 Tech Preview
+keywords: Docker, WSL, WSL 2, Tech Preview, Windows Subsystem for Linux
+title: Docker Desktop WSL 2 Tech Preview
+toc_min: 1
+toc_max: 2
+---
+
+# Overview
+
+Welcome to Docker Desktop WSL 2 Tech Preview. This Tech Preview introduces support to run Docker Desktop with WSL 2. We really appreciate you trialing this Tech Preview. Your feedback is very important to us. Please let us know your feedback by creating an issue in the [Docker Desktop for Windows GitHub](https://github.com/docker/for-win/issues) repository and adding the **WSL 2** label.
+
+WSL 2 introduces a significant architectural change as it is a full Linux kernel built by Microsoft, allowing Linux containers to run natively without emulation. With Docker Desktop WSL 2 Tech Preview, users can access Linux workspaces without having to maintain both Linux and Windows build scripts.
+
+Docker Desktop also leverages the dynamic memory allocation feature in WSL 2 to greatly improve the resource consumption. This means, Docker Desktop only uses the required amount of CPU and memory resources, enabling CPU and memory-intensive tasks such as building a container to run much faster.
+
+Additionally, with WSL2, the time required to start a Docker daemon after a cold start is significantly faster. It takes less than 2 seconds to start the Docker daemon when compared to tens of seconds in the current version of Docker Desktop.
```

Comment:
It's written as WSL2 here, but WSL 2 in the previous paragraph.

Reply:
another good catch. I'll make the change to WSL 2.